src/app/core/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -8,9 +8,9 @@ import { PageInfo } from 'src/app/shared/services/current-content.service';
 })
 export class BreadcrumbComponent {
 
-  @Input() pageInfo: PageInfo|null
+  @Input() contentBreadcrumb?: ContentBreadcrumb
```

Comment:
why did you put the `contentBreadcrumb` as optional instead of letting `|null` ?

Reply:
Because the breadcrumb attribute of ContentInfo is optional and not null.